lib/LightListView.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:huenicorn/LightRowView.dart';
-import 'package:huenicorn/light.dart';
+import 'package:huenicorn/hue/light.dart' show Light;
```

Comment:
`Light` is the only entity in "light.dart", so I don't see the reason to use `show Light` here.